src/Autohide.js

```diff
@@ -182,10 +186,55 @@ define(function (require, exports, module) {
         exposeDelay = delay;
     }
 
+    function showDelayDialog() {
+        var Dialog = Dialogs.showModalDialog(
+            brackets.DIALOG_ID_SAVE_CLOSE,
+            Strings.AUTOHIDE_DELAY_MODAL_TITLE,
+            Mustache.render(sliderTemplate, {
+                exposeDelay: exposeDelay,
+                millisecondsLabel: Strings.AUTOHIDE_DELAY_MODAL_MS_LABEL
+            })
+        );
+
+        var $slider = $("#outline-delay-slider");
+        var $inputbox = $("#outline-delay-inputbox");
+        var inputboxCurrentValue = $inputbox.val();
+
+        Dialog.done(function (buttonId) {
+            if (buttonId === "ok") {
+                prefs.set("autohideDelay", $slider.val());
+            }
+        });
+
+        // Slider change handler
+        $slider.on("input", function () {
+            inputboxCurrentValue = $slider.val();
+            $inputbox.val(inputboxCurrentValue);
+        });
+
+        // Inputbox handler
+        $inputbox.keyup(function (ev) {
+            if ((ev.keyCode < 48 || ev.keyCode > 57) && ev.keyCode !== 8) {
```

Comment:
Are there any keyCode constants that can be used here?
If not, please add comments so a reader doesn't have to look up the codes.